src/pages/index.html

```diff
@@ -29,14 +32,25 @@
                 <source src="../public/video/head_vid.mp4" type="video/mp4">
                 Your browser does not support the video tag.
             </video>
+            
+            <!-- hero text-->
+            <div class ="hero-text">

```

Comment:
Try to follow the convention we agreed upon: `homepage-video-text-`  maybe might be better and more descriptive. You could potentially have this as an `aside`  as well instead of a plain `div`.